src/view/sorting.js

```diff
@@ -8,33 +9,49 @@ const createSortingTemplate = () => {
   return `<form class="trip-events__trip-sort  trip-sort" action="#" method="get">
   <div class="trip-sort__item  trip-sort__item--day">
     <input id="sort-day" class="trip-sort__input  visually-hidden" type="radio" name="trip-sort" value="sort-day" checked>
-    <label class="trip-sort__btn" for="sort-day">Day</label>
+    <label class="trip-sort__btn" for="sort-day" data-sort-type="${SortTypes.DEFAULT}">Day</label>
```

Comment:
```suggestion
    <label class="trip-sort__btn" for="sort-day" data-sort-type="${SortTypes.DAY}">Day</label>
```
Там же нет `DEFAULT`.